numpy/core/einsumfunc.py

```diff
@@ -1051,6 +1119,8 @@ def einsum(*operands, **kwargs):
     array([[10, 28, 46, 64],
            [13, 40, 67, 94]])
 
+    Writeable returned arrays:
+
     >>> # since version 1.10.0
```

Comment:
May as well move this comment to the line above

Reply:
done